be/src/connector/jdbc_connector.cpp

```diff
@@ -48,9 +48,16 @@ const TupleDescriptor* JDBCDataSourceProvider::tuple_descriptor(RuntimeState* st
 // ================================
 
 static std::string get_jdbc_sql(const Slice jdbc_url, const std::string& table, const std::vector<std::string>& columns,
-                                const std::vector<std::string>& filters, int64_t limit) {
+                                const std::vector<std::string>& filters, const std::vector<std::string>& session_variable_hints,
+                                int64_t limit) {
     std::ostringstream oss;
     oss << "SELECT";
+    // for mysql, we can set session variable hints in select statement
+    if (jdbc_url.starts_with("jdbc:mysql")) {
+        for (size_t i = 0; i < session_variable_hints.size(); i++) {
+            oss << " " << session_variable_hints[i];
```

Comment:
what that hint would be like?
like "/*+ key=value */" ?

Reply:
Added a comment for clarity